contracts/ResilientOracle.sol

```diff
@@ -44,6 +45,8 @@ isValid = anchorRatio <= upperBoundAnchorRatio && anchorRatio >= lowerBoundAncho
  * oracle to be stagnant and treat it like it's disabled.
  */
 contract ResilientOracle is PausableUpgradeable, AccessControlledV8, ResilientOracleInterface {
+    using Transient for bytes32;
```

Comment:
Do we need this? We are calling the library functions this way: `Transient.function(...)`, not from a `bytes32` variable. So, I think we don't need to "link" the library to any data type.

Alternatively, if we maintain this statement (`using Transient for bytes32;`), we could remove the first parameter from the library functions (the cache slot), and call the functions this way: `CACHE_SLOT.function(...)`

Reply:
https://github.com/VenusProtocol/oracle/pull/239/commits/27267d8522b243173280f7eff17278474ced5aa8